DEMO/LEDsetup.py

```diff
@@ -68,4 +61,3 @@
 Khaki = (240, 150, 60)
```

Comment:
`bibliopixel.util.colors` has all these names and many more, so you can just say

    from bibliopixel.util import colors
    
    color_list = [colors.Red, colors.Green, colors.Yellow]